app/server/handlers/render-layout.js

```diff
@@ -38,22 +41,21 @@ export const getCriticalCss = async () => {
 export async function renderLayout(res, params) {
   const chunk = params.shell ? null : allChunks[getChunkName(params.pageType)];
   const { gtmId, gaId, cdnImage, isGtmEnable, isGaEnable } = getConfig(params.store.getState());
-
   res.render(
     "pages/layout",
     Object.assign(
       {
         assetPath: assetPath,
         content: "",
         cssContent: cssContent,
-        criticalCss: getCriticalCss(),
+        criticalCss: await getCriticalCss(),
```

Comment:
I dont think await is needed. If so can you assign it to a variable.

Reply:
done